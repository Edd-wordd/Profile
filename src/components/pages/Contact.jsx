import React, { Suspense, lazy } from 'react'
import Spinner from '../../components/spinners/Spinner'

const ContactForm = lazy(() => import('../../components/forms/ContactForm'))

function Contact() {
  return (
    <>
      <Suspense fallback={<Spinner />}>
        <ContactForm />
      </Suspense>
    </>
  )
}

export default Contact
