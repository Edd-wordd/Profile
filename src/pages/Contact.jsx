import React from 'react'
import ContactForm from '../components/ContactForm'
import Footer from '../components/Footer'
import GetInTouch from '../components/GetInTouch'

function Contact() {
  return (
    <>
      <ContactForm />\
      <GetInTouch />
      <Footer />
    </>
  )
}

export default Contact
