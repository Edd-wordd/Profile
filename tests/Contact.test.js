import React from 'react'
import testComponent from './testComponent'
import Contact from '../src/components/pages/Contact'

testComponent(<Contact />, 'Contact')
