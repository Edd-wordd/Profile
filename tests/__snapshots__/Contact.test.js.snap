// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render Contact 1`] = `"Component"`;
